openapi.yaml

```diff
@@ -626,6 +660,84 @@ paths:
 
 components:
   schemas:
+    AggregatorStatusMessage:
+      description: Represents the information related to the aggregator status
+      type: object
+      additionalProperties: false
+      required:
+        - epoch
+        - cardano_era
+        - mithril_era
+        - cardano_node_version
+        - aggregator_node_version
+        - protocol
+        - next_protocol
+        - total_signers
+        - total_next_signers
+        - total_stakes_signers
+        - total_next_stakes_signers
+        - total_cardano_spo
+        - total_cardano_stake
+      properties:
+        epoch:
+          $ref: "#/components/schemas/Epoch"
+        cardano_era:
+          description: Cardano era
+          type: string
+        mithril_era:
+          description: Mithril era
+          type: string
+        cardano_node_version:
+          description: Version of the Cardano node
+          type: string
+        aggregator_node_version:
+          description: Version of the Aggregator node
+          type: string
+        protocol:
+          $ref: "#/components/schemas/ProtocolParameters"
+        next_protocol:
+          $ref: "#/components/schemas/ProtocolParameters"
+        total_signers:
+          description: The number of signers for the current epoch
+          type: integer
+          format: int64
+        total_next_signers:
+          description: The number of signers that will be able to sign on the next epoch
+          type: integer
+          format: int64
+        total_stakes_signers:
+          description: The total stakes of signers for the current epoch
+          type: integer
+          format: int64
+        total_next_stakes_signers:
+          description: The total stakes of signers for the next epoch
+          type: integer
+          format: int64
+        total_cardano_spo:
+          description: The number of Cardano SPOs
+          type: integer
+          format: int64
+        total_cardano_stake:
+          description: The total stake in Cardano
+          type: integer
+          format: int64
+      examples:
+        {
+          "epoch": 329,
+          "cardano_era": "conway",
```

Comment:
```suggestion
          "cardano_era": "Conway",
```

Reply:
The values returned by the `EraReader` for the Mithril and Cardano eras from the `ChainObserver` don’t have consistent casing. Should I handle capitalizing the first letter in the `EpochService` for both values?

{
  "epoch": 85,
  "cardano_era": "Babbage",
  "mithril_era": "thales",
  ...
}

Reply:
Let's stick to the real output :+1: